src/plugins/topic_viewer/TopicViewer.qml

```diff
@@ -0,0 +1,252 @@
+            {
+                // emit drop event to notify the DropArea (must manually)
+                parent.Drag.drop();
+            }
+
+            hoverEnabled: true
+            propagateComposedEvents: true
+            // make the cursor with a drag shape at the plottable items
+            cursorShape: (model === null) ?  Qt.ArrowCursor : (model.plottable) ?
+                                                Qt.DragCopyCursor : Qt.ArrowCursor
+
+            onClicked: {
+                // change the selection of the tree by clearing the prev, select a new one
+                tree.selection.select(styleData.index,ItemSelectionModel.ClearAndSelect)
+
+                // set the selection index to the index of the clicked item (must set manually)
+                tree.selection.setCurrentIndex(styleData.index,ItemSelectionModel.ClearAndSelect)
+
+                // the currentIndex of the tree.selection is not the same
+                // of the tree.currentIndex, so set the tree.currentIndex.
+                // this is the way to access it as it is read-only
+                tree.__currentRow = styleData.row
+
+                // set the focus to the selected item to receive the keyboard events
+                // this is useful to enable navigating with keyboard from the right position
+                item.forceActiveFocus();
+
+                tree.expandCollapseMsg(tree.currentIndex);
+            }
+        }
+
+        Image {
+            id: icon
+            source: "plottable_icon.svg"
+            height: itemHeight * 0.6
+            width: itemHeight * 0.6
+            y : itemHeight * 0.2
+            visible: (model === null) ? false : model.plottable
+        }
+
+        Text {
+            id : field
+            text: (model === null) ? "" : model.name
+            color: (Material.theme == Material.Light || styleData.selected) ?
+                    Material.color(Material.Grey, Material.Shade800):
+                    Material.color(Material.Grey, Material.Shade400);
+
+            font.pointSize: 12
+            anchors.leftMargin: 5
+            anchors.left: icon.right
+            y: icon.y
+        }
+
+        ToolTip {
+            id: tool_tip
+            delay: 1000
+            timeout: 2000
+            text: (model === null) ? "Type ?" : "Type: " + model.type;
+            visible: dragMouse.containsMouse
+            y: -itemHeight
+            x: dragMouse.mouseX
+            enter: null
+            exit: null
+        }
+    }
+
+    property int y_pos: 0
+
+    style: TreeViewStyle {
+        branchDelegate: Rectangle {
+          height: itemHeight
+          width: itemHeight
+          color: "transparent"
+          Image {
+            id: branchImage
+            fillMode: Image.Pad
+            anchors.right: parent.right
+            anchors.verticalCenter: parent.verticalCenter
+            sourceSize.height: itemHeight * 0.5
+            sourceSize.width: itemHeight * 0.5
```

Comment:
```suggestion
            sourceSize.width: itemHeight * 0.4
```

The icons on this plugin are bigger than the ones on the Entity Tree now. How about giving them the same size, multiplying by 0.4 instead? It will look like this:

![iconsize_plot](https://user-images.githubusercontent.com/5751272/88335759-acb14880-cce8-11ea-8f8c-9da0e70b68c5.png)